### PR TITLE
Add extra conf for cinder and ceph

### PR DESCRIPTION
**Pull request by ktibi**

### CODE REVIEW

File: ansible/kolla-openstack.yml [hunk 1]
```diff
@@ -192,6 +192,8 @@
       kolla_inspector_dhcp_pool_end: "{{ inspection_net_name | net_inspection_allocation_pool_end }}"
       kolla_inspector_default_gateway: "{{ inspection_net_name | net_inspection_gateway or inspection_net_name | net_gateway }}"
       # Extra free-form user-provided configuration.
+      kolla_extra_ceph: "{{ kolla_extra_config.ceph | default }}"
+      kolla_extra_cinder: "{{ kolla_extra_config.ciner | default }}"
```
Comment (markgoddard) [inline ansible/kolla-openstack.yml]: s/ciner/cinder/

Comment (markgoddard) [reply ansible/kolla-openstack.yml]: Also need to add these on L99

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: ansible/kolla-openstack.yml (hunk 1 above)
Comment (ktibi) [reply ansible/kolla-openstack.yml]: done

**Review by markgoddard — APPROVED** [review]
Looks better. Tested?

**Comment by ktibi** [comment]
yes tested and work

**Pull request merged**